hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/package.json

```diff
@@ -63,109 +56,23 @@
     ]
   },
   "devDependencies": {
-    "@types/react-plotly.js": "^2.2.4",
+    "@types/react": "16.8.15",
+    "@types/react-dom": "16.8.4",
+    "@types/react-router-dom": "^4.3.5",
```

Comment:
Does React dependencies supposed to be moved from `dependencies` to `devDependencies`?

Reply:
Yes Ivan, so `@types/...` provide typedefs.
We should include them in the devDependencies only if we are exporting those types in the final build as well.
For example if we are creating a custom library, and we need to provide the types to our end users to use the libs then it should mandatorily be in dependencies.

But in this case since we are only consuming the types as a part of the development and not required for the final build product we can safely move them out I believe.